ext/json/json.c

```diff
@@ -42,26 +42,7 @@ PHP_JSON_API zend_class_entry *php_json_exception_ce;
 
 PHP_JSON_API ZEND_DECLARE_MODULE_GLOBALS(json)
 
-/* {{{ arginfo */
-ZEND_BEGIN_ARG_INFO_EX(arginfo_json_encode, 0, 0, 1)
-	ZEND_ARG_INFO(0, value)
-	ZEND_ARG_INFO(0, options)
-	ZEND_ARG_INFO(0, depth)
-ZEND_END_ARG_INFO()
-
-ZEND_BEGIN_ARG_INFO_EX(arginfo_json_decode, 0, 0, 1)
-	ZEND_ARG_INFO(0, json)
-	ZEND_ARG_INFO(0, assoc)
-	ZEND_ARG_INFO(0, depth)
-	ZEND_ARG_INFO(0, options)
-ZEND_END_ARG_INFO()
-
-ZEND_BEGIN_ARG_INFO(arginfo_json_last_error, 0)
-ZEND_END_ARG_INFO()
-
-ZEND_BEGIN_ARG_INFO(arginfo_json_last_error_msg, 0)
-ZEND_END_ARG_INFO()
-/* }}} */
+#include "json_arginfo.h"
```

Comment:
Nit: can we move this one to the top of this file together with the other includes?

Reply:
Happy to if that's the convention, I was just following how @nikic did it here: https://github.com/nikic/php-src/commit/33886f710cd45fd883d85558197e2df4eaba9ea1

Reply:
Let's make it a convention, what do you think? Just like we declare our `use Foo\Bar;` in PHP in the top of the files.

Reply:
Agree that top of the file is better.